search.json

```diff
@@ -0,0 +1,14 @@
+---
+layout: null
+---
+[
+    {% for post in site.posts %}
+        {
+        "title"    : "{{ post.title | escape }}",
+        "category" : "{{ post.category }}",
+        "tags"     : "{{ post.tags | join: ', ' }}",
+        "url"      : "{{ site.baseurl }}{{ post.url }}",
+        "content": "{{ page.content | strip_html | strip_newlines | remove_chars | escape }}"
+        } {% unless forloop.last %},{% endunless %}
```

Comment:
wow, I didn't know you could do that

Reply:
we don't need these files now, actually! :)